inspirehep/utils/latex.py

```diff
@@ -22,6 +22,7 @@
 
 from __future__ import absolute_import, division, print_function
 
+import latexcodec  # noqa: F401
```

Comment:
Yikes, is this package really modifying state at import time? That sounds like a really bad idea. Is there another package that we can use instead with a more sensible API?

Reply:
And if instead the only option we have is this package, can we at least ensure that this import doesn't happen deep in the code, where it might happen for some code paths but not others?

Reply:
@michamos investigated for a while and this seemed the best one. 

> can we at least ensure that this import doesn't happen deep in the code

What do you mean?

Reply:
it's doing only [`codecs.register`](https://github.com/mcmtroffaes/latexcodec/blob/develop/latexcodec/codec.py#L70-L75), on import, which is needed in order to plug into `str.encode` etc. Is that OK or should it be imported differently?